org.kde.okular.json

```diff
@@ -287,6 +287,23 @@
             }
          ]
       },
+      {
+         "name": "okular-wrapper",
```

Comment:
And it doesn't need to be a different module, you can put it directly under okular sources with:
`{
                    "type": "script",
                    "dest-filename": "okular-wrapper",
                    "commands": [
                        "exec env TMPDIR=$XDG_CACHE_HOME /app/bin/okular \"$@\""
                    ]
                }`

Reply:
You could also add `--env=TMPDIR=/var/tmp` to `finish-args`.

Reply:
You also might want to comment on https://github.com/flatpak/flatpak/issues/3438.

Reply:
`/var/tmp` won't work. I commented extensively in mentioned topic already.